cmd/slsa-provenance/cli/files_test.go

```diff
@@ -0,0 +1,180 @@
+package cli_test
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/philips-labs/slsa-provenance-action/cmd/slsa-provenance/cli"
+)
+
+func TestGenerateFilesCliOptions(t *testing.T) {
+	_, filename, _, _ := runtime.Caller(0)
+	rootDir := path.Join(path.Dir(filename), "../../..")
+	provenanceFile := path.Join(rootDir, "bin/unittest-provenance.json")
+
+	testCases := []struct {
+		name      string
+		err       error
+		arguments []string
+	}{
+		{
+			name:      "without commandline flags",
+			err:       cli.RequiredFlagError("artifact-path"),
+			arguments: make([]string, 0),
+		},
+		{
+			name: "only providing --artifact-path",
+			err:  cli.RequiredFlagError("github-context"),
+			arguments: []string{
+				"--artifact-path",
+				path.Join(rootDir, "bin/slsa-provenance"),
+			},
+		},
+		{
+			name: "without -runner-context",
+			err:  cli.RequiredFlagError("runner-context"),
+			arguments: []string{
+				"--artifact-path",
+				path.Join(rootDir, "bin/slsa-provenance"),
+				"--github-context",
+				githubContext,
+				"--output-path",
+				provenanceFile,
+			},
+		},
+		{
+			name: "invalid --artifact-path",
+			err:  fmt.Errorf("failed to generate provenance: resource path not found: [provided=non-existing-folder/unknown-file]"),
```

Comment:
Can the ``non-existing-folder/unknown-file`` be a constant or similar? I see it being reused multiple times. Might be convenient. 
If you do not think so, ignore this comment. 
The same goes for the other error messages.